File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/media/Asset.java
package com.BlizzardArmory.model.warcraft.equipment.media;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * The type Asset.
 */
public class Asset {

    @SerializedName("key")
    @Expose
    private String key;
    @SerializedName("value")
    @Expose
    private String value;

    /**
     * Gets key.
     *
     * @return the key
     */
    public String getKey() {
        return key;
    }

    /**
     * Sets key.
     *
     * @param key the key
     */
    public void setKey(String key) {
        this.key = key;
    }

    /**
     * Gets value.
     *
     * @return the value
     */
    public String getValue() {
        return value;
    }

    /**
     * Sets value.
     *
     * @param value the value
     */
    public void setValue(String value) {
        this.value = value;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/media/Asset.kt
package com.BlizzardArmory.model.warcraft.equipment.media

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * The type Asset.
 */
data class Asset(

        @SerializedName("key")
        @Expose
        var key: String,

        @SerializedName("value")
        @Expose
        var value: String

)